index.html

```diff
@@ -1,117 +1,283 @@
+                            </li>
+                            <li>
+                              Be kind and considerate to your fellow hackers and our volunteers. We're all here to have fun! By participating in the hackathon, you agree to abide by <a href="https://hackcodeofconduct.org/" target="_blank">this Code of Conduct</a>.
+                            </li>
+                          </ul>
+                        </div>  
+                        </li>                   
+ 
+                     </ul>                                            
+                </div>
+     </section>
+     <!-- RULES  SECTION END-->
+      <section id="contact-sec"  >
+        <div class="overlay">           
+          <div class="container basic-set">
+            <div class="row " >
+              <div class="col-md-4 col-sm-4" >
+                <h3>OUR LOCATION</h3>
+                <br />
+                <h4>William Gates Building, 15 J.J. Thomson Ave, Cambridge CB3 0FD</h4>
+                <h4>Cambridge, Cambridgeshire</h4>
+                <h4>United Kingdom</h4>
+              <br />
+               <strong>E-mail :</strong> team@hackersatcambridge.com
+                 <br /><br />
+                </div>
+                <div class="col-md-4 col-sm-4">
+                   <h3>WE ARE SOCIAL</h3>
+                    <div class="social">
+                      <a href="https://www.facebook.com/hackersatcambridge/" style="background-color:#3b5998;"><i class="fa fa-facebook-square " ></i>  Facebook </a>
+                      <a href="https://twitter.com/hackersatcam" style="background-color:#55acee;"><i class="fa fa-twitter-square " ></i>  Twitter </a>
+                    </div>
+                   <h6> Drop us a message at any point, regarding the hackathon of just general Hackers at Cambridge matters! </h6>
+             
+              
+                </div>
+               
+               <div class="col-md-4 col-sm-4 subscribe"  >
+                   
+                   <h2 >REGISTER FOR CHARITECH HERE</h2>
+                   <br />
+                   <p>
+                     If you haven't already, you can register for the hackathon now. <br>
+                     The button below will redirect you to the event page on EventBrite.
+                   </p>
+                  
+                   <hr />
+                    <br />
+                   <div class="form-group">
+                       <a href="https://www.eventbrite.co.uk/e/charitech-tickets-32356981603?aff=es2" class="btn btn-primary btn-lg btn-block black-btn"> REGISTER FOR CHARITECH </a> 
+                   </div>
+               </div>
             </div>
           </div>
         </div>
-      </div>
-      </div>
+      </section>
+    <!-- CONTACT  SECTION END-->
 
+    <!-- JAVASCRIPT FILES PLACED AT THE BOTTOM TO REDUCE THE LOADING TIME  -->
+    <!-- CORE JQUERY  -->
+    <script src="assets/js/jquery-1.11.1.js"></script>
```

Comment:
I see you've added jQuery and Bootstrap and some various plugins as dependencies. This is great for speeding up development time but not everything comes for free. In particular, these files are pretty big, leading us to load in about 400kb of JS for a pretty small website. 

There are some easy wins here. We can use the minified versions of these libraries. We can also use a more recent version of jQuery (2/3+) which shaves a lot of file size. We can also get more speed, leverage more browser caching and let us not commit dependencies to our repo by using a cdn like cdnjs.com

- https://cdnjs.com/libraries/twitter-bootstrap
- https://cdnjs.com/libraries/jquery